Dockerfile.fastapi

```
@@ -0,0 +1,115 @@
+# Stage 2: Builder - Install runtime dependencies
+# Stage 3: Runtime - Minimal runtime environment
+
+# ============================================================================
+# STAGE 1: Libdoc Builder - Generate library documentation
+# ============================================================================
+FROM python:3.12-slim AS libdoc-builder
+
+WORKDIR /build
+
+# Install Node.js 20.x for Browser library initialization
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)